sale_dost/models/sale_dost.py

```diff
@@ -1,20 +1,34 @@
 # -*- coding: utf-8 -*-
 # See LICENSE file for full copyright and licensing details.
 
-from openerp import fields, models,api
+from odoo import fields, models, api
+import base64
 
 
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
+    
+    @api.multi
+    def picking_state(self):
+        for rec in self:
+            rec.shipped = False
```

Comment:
2 times write thaay e khotu. res  = False karo and last ma common rec.shipped = res karo